script.js

```diff
@@ -2,3 +2,83 @@ var main = function (input) {
   var myOutputValue = 'hello world';
   return myOutputValue;
 };
+// set global var//
+var win = 0;
+var lose = 0;
+var draw = 0;
+var troll = 0;
```

Comment:
What is `troll`? Please explain this in a comment. Please be liberal about comments- use comments wherever the code may be confusing to anyone.